tests/test_service.py

```diff
@@ -0,0 +1,153 @@
+try:
+    from unittest.mock import create_autospec, Mock, patch, MagicMock
+except ImportError:
+    from mock import create_autospec, Mock, patch, MagicMock
+
+from reportportal_client.service import _convert_string, _list_to_payload, \
+    uri_join, _get_id, _get_msg, _get_data, _get_json, _get_messages, \
+    ReportPortalService
+
+import unittest
+from datetime import datetime
+
+
+class TestServiceFunctions(unittest.TestCase):
+    """
+    Test for main functions in service.py
```

Comment:
One-line docstring should fit on one line with quotes.  https://www.python.org/dev/peps/pep-0257/#one-line-docstrings